CONTRIBUTING.md

```diff
@@ -122,6 +122,19 @@ output if necessary:
 ./gradlew check
 ```
 
+All submitted GraalVM reachability metadata, if the MBean-related part is unnecessary, should actively remove the
```

Comment:
We should fix the root cause in Native Image. Could you be more precise about what is failing if we introduce MBean metadata?

Reply:
- As an additional topic, can I bring the following reference links to this document, namely https://github.com/oracle/graalvm-reachability-metadata/pull/162 , https://github.com/oracle/graalvm-reachability-metadata/pull/161 and https://github.com/oracle/graalvm-reachability-metadata/pull/113 ?

Reply:
- I tried to describe this process in the latest commit. But since this is currently only happening on the Spring Boot side, I'm not too sure about the more specific behavior. 

- I haven't personally investigated AOT on Spring Boot 3.x yet. Because I'm blocked by other behavior at https://github.com/oracle/graalvm-reachability-metadata/pull/154.